### ISSUE TITLE
It does not iterate over the batches

### ISSUE THREAD

**Issue by SRKH** [NeuroCompLab-psu/SNN-Conversion#2]
The code does not iterate over the test batches. To fix it, the line 131 of Conversion.py:

`        args.eval_size = len(dataloader)
`

should be changed to:

`        args.eval_size = len(val_loader)
`

By fixing it, and running the following command:

`python conversion.py --job-dir cifar100_test --gpu --dataset cifar100 --data . --percentile 99.9 --norm 3500 --arch vgg15ab --model cifar100_model_best.pth.tar`

the final accuracy for the full-precision model on cifar100 (available in cifar100_model_best.pth.tar) was 60.48% and not 64.9%:

>  ....
> 97% 97/100 [04:19<00:08,  2.70s/it]batch
>  98% 98/100 [04:21<00:05,  2.70s/it]batch
>  99% 99/100 [04:24<00:02,  2.70s/it]batch
> 100% 100/100 [04:27<00:00,  2.67s/it]
> Figure size 640x480 with 1 Axes
> SNN accuracy: 60.48
> Clock time used: 267.3109 ms.

**Comment by lusensama** [comment]
Hi, Thank you for finding out the bug on eval_size! I will fix that soon.

But for the script that you are running, the default timesteps taken is only 80. Have you tried increasing the timesteps to be simulated as well? It should reach around 62% eventually.

Please also note that the accuracy values stated next to the pretrained models are the ANN accuracy, not the converted SNN accuracy.

Hope that clears the confusion.
